minigotchi/deauth.cpp

```diff
@@ -103,16 +104,7 @@ bool Deauth::broadcast(uint8_t *mac) {
   return true;
 }
 
-void Deauth::printMac(uint8_t *mac) {
-  for (int i = 0; i < 6; i++) {
-    Serial.print(mac[i], HEX);
-    if (i < 5) {
-      Serial.print(":");
-    }
-  }
-
-  Serial.println();
-}
+void Deauth::printMac(uint8_t *mac) { Serial.println(printMacStr(mac)); }
```

Comment:
`printMac` just prints it to Serial so we couldn't use that one on line 329 since it's passing it into the display method, but we could get rid of `printMac` and replace line 327 with `Serial.println(printMacStr)`

Reply:
Oh I misunderstood what you meant, sorry. 
I'll move it in there

Reply:
either way it doesn't matter, i just have it to have it